engine/openapi.ftl

```diff
@@ -504,14 +508,16 @@
                 [#local Authorizer = value.Authorizer!{} ]
                 [#if Authorizer?has_content]
                     [#switch Authorizer.Type]
+                        [#-- Default validity expression is for bearer based JWT --]
                         [#case "token"]
                             [#local scheme +=
                                 {
                                     "x-amazon-apigateway-authorizer": {
                                         "type": "token",
                                         "authorizerUri" : formatLambdaArnUsingStageVariable(context, Authorizer.Variable),
-                                        "authorizerResultTtlInSeconds" : Authorizer.TTL ! 300
-                                    }
+                                        "authorizerResultTtlInSeconds" : Authorizer.TTL ! 300,
+                                        "identityValidationExpression" : Authorizer.ValidityExpression ! r"^[ \t]*bearer[ \t]+[A-Za-z0-9+\/=]+\.[A-Za-z0-9+\/=]+\.[A-Za-z0-9+\/=]+[ \t]*$"
```

Comment:
Shoudn't these be defaults on the configuration schema above instead of hard coded? We can't document them where they are now

Reply:
Fair point - I just copied what was done for the TTL. If I add a default to the schema, then it will always set the attribute which is probably advantageous anyway

Reply:
Note that these settings are on the integrations file so atm, it won't show up in the component configuration. Maybe we should move this to a reference of some type.